scripts/utils/docs-content-tools/handlers.js

```diff
@@ -0,0 +1,142 @@
+const glob = require('glob');
+const frontmatterGH = require('@github-docs/frontmatter');
+const fs = require('fs');
+
+/**
+ * Adds slashes to beginning and end of directory if missing
+ * @param {String} directory - The directory to add slashes to
+ * @returns {String} The directory with slashes added if necessary
+ */
+const formatDirectory = (directory) => {
+  !directory.startsWith('/') && (directory = `/${directory}`);
+  !directory.endsWith('/') && (directory = directory.concat('/'));
+  return directory;
+};
+
+/**
+ * Gets slug from path, relative to contents folder
+ * @param {String} filepath - The full filepath of the file
+ * @returns {String} The slug of the filepath
+ */
+const getSlugFromPath = (filepath) => {
+  return filepath.match(/content(.*?).mdx/)[1];
+};
+
+/**
+ * Reads a file and separates the content and frontmatter data
+ * @param {String} path - The full filepath of the file
+ * @returns {Object} Original path, slug from path, content, frontmatter data
+ */
+const separateData = async (path) => {
+  const file = fs.readFileSync(path, 'utf8');
+  const slug = getSlugFromPath(path);
+  const separateData = await frontmatterGH(file);
```

Comment:
`frontmatterGH` is synchronous and therefore does not need to be `await`ed.

```suggestion
  const separateData = frontmatterGH(file);
```